runtime/doc/eval.txt

```diff
@@ -4061,9 +4061,8 @@ jobstop({job})						{Nvim} *jobstop()*
 		Stop a job created with |jobstart()| by sending a `SIGTERM`
 		to the corresponding process. If the process doesn't exit
 		cleanly soon, a `SIGKILL` will be sent. When the job is
-		finally closed, a |JobActivity| event will trigger with
-		`v:job_data[0]` set to `exited`.  See |job-control| for more
-		information.
+		finally closed, the exit handler provided to |jobstart()| or
+		|termopen()| will be run.  See |job-control| for more information.
```

Comment:
Could you break the line after the period? It seems to be commonly done for sentences matching the scheme `See |help-doc| for more information`.